src/Servers/IIS/AspNetCoreModuleV2/gtest/gtest.vcxproj

```diff
@@ -179,4 +181,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
+</Project>
```

Comment:
Why did you eat the newline?

Reply:
It just automatically did when I saved the file in VS.